### PR TITLE
ubuntu-16.04-cross-arm64:Install binutils-aarch64-linux-gnu

### PR DESCRIPTION
**Pull request by sdmaclea**
@janvorli @MichaelSimons Towards fixing https://github.com/dotnet/coreclr/issues/16890

### CODE REVIEW

File: src/ubuntu/16.04/cross/arm64/Dockerfile [hunk 1]
```diff
@@ -1,3 +1,9 @@
 FROM microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-crossdeps
 
+# Install binutils-aarch64-linux-gnu
```
Comment (janvorli) [inline src/ubuntu/16.04/cross/arm64/Dockerfile]: Looking at how we handle that for arm, it seems this rather belongs to the ubuntu-16.04-crossdeps Dockerfile. We install the binutils-arm-linux-gnueabihf in there, so it seems this should be located next to that.

@MichaelSimons what do you think?

Comment (MichaelSimons) [reply src/ubuntu/16.04/cross/arm64/Dockerfile]: As long as `binutils-aarch64-linux-gnu` isn't needed for building the rootfs, I think adding it to the arch specific cross image is preferred.  The primary benefit is that it keeps the arch specific components out of the crossdeps image which is being shared across arch images.  If possible `binutils-arm-linux-gnueabihf` should get pulled out of the crossdeps image as part of #16 and moved into the arm cross image.

Comment (sdmaclea) [reply src/ubuntu/16.04/cross/arm64/Dockerfile]: `binutils-aarch64-linux-gnu` shouldn't be required for building the rootfs

**Comment by MichaelSimons** [comment]
The following images were built as a result of this PR:

microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-d809634-20180312171656
microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-debpkg-d770b8b-20180312171656
microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-coredeps-d770b8b-20180312171656
microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-crossdeps-d770b8b-20180312171656
microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-arm64-d809634-20180312171656
microsoft/dotnet-buildtools-prereqs:ubuntu-16.04-cross-d809634-20180312171656

**Pull request merged**